client/my-sites/plugins/marketplace/marketplace-test/admin-menu-fetch.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { Button, Card } from '@automattic/components';
+import { useSelector, useDispatch } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import useSiteMenuItems from 'calypso/my-sites/sidebar-unified/use-site-menu-items';
+import { getIsRequestingAdminMenu } from 'calypso/state/admin-menu/selectors';
+import { requestAdminMenu } from 'calypso/state/admin-menu/actions';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { level1ObjectMap } from 'calypso/my-sites/plugins/marketplace/marketplace-test';
+import CardHeading from 'calypso/components/card-heading';
+
+export default function AdminMenuFetch(): JSX.Element {
+	const menuItems = useSiteMenuItems();
+	const isRequestingMenu = useSelector( getIsRequestingAdminMenu );
+	const selectedSiteId = useSelector( getSelectedSiteId );
+	const dispatch = useDispatch();
+
+	const { url: postsPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'edit-php' ) ?? {};
+
+	const { url: yoastSeoPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'wpseo_dashboard' ) ?? {};
+
+	useEffect( () => {
+		if ( ! postsPageUrl || ! yoastSeoPageUrl ) {
+			selectedSiteId && dispatch( requestAdminMenu( selectedSiteId ) );
+		}
+	}, [ dispatch, postsPageUrl, selectedSiteId, yoastSeoPageUrl ] );
+
+	return (
+		<Card>
+			<Button onClick={ () => dispatch( requestAdminMenu( selectedSiteId ) ) }>Refresh</Button>
```

Comment:
Could we add a `@TODO` or some similar comment to ensure we don't forget to translate those non-localized strings once this is final?

Reply:
We don't need to translate this component because a user will never see it.